src/docs/index.jsx

```diff
@@ -302,7 +332,17 @@ class Demo extends Component {
                   required
                 />
               </span>
+              {reportMode === 'create' && (
```

Comment:
create a variable `const isCreateMode = reportMode === 'create'` and use `isCreateMode` across. Let's reduce the comparisons.

Reply:
donne